ad-blocker-update.rst

```diff
@@ -8,7 +8,7 @@ Update on Ad Blocking and Acceptable Ads
 ========================================
 
 A few weeks ago, we shared about the
-:doc:`challenge ad blocking presented to our sustainability <ads-and-adblocking>`
+:doc:`challenge ad blocking presented to our sustainability </ads-and-adblocking>`
```

Comment:
For some reason sphinx gives a warning that it can locate the document when it doesn't start with `/`, so I had to update those.